src/Generator/Passes/CheckAbiParameters.cs

```diff
@@ -68,7 +68,11 @@ public override bool VisitFunctionDecl(Function function)
 
             // Deleting destructors (default in v-table) accept an i32 bitfield as a
             // second parameter in MS ABI.
-            if (method != null && method.IsDestructor && Context.ParserOptions.IsMicrosoftAbi)
+            var @class = method != null ? method.Namespace as Class : null;
```

Comment:
Needlessly complicated - if the method is not null, then its name-space is always a class, just cast it.